### ISSUE TITLE
Searching for Git on Windows

### ISSUE THREAD

**Issue by 50Wliu** [smashwilson/merge-conflicts#95]
See comments on #88 for context.
Basically, my Git installation is at `%appdata%/../Local/Programs/Git/bin/git.exe`.  In addition, if you are specifying a path, I'm pretty sure that the executable would need to be `git.exe`, not `git` (which would work through the PATH).
Will try to test the `%appdata%` thing out when I have merge conflicts.

I know nothing about Coffeescript, so no PR from me :(

**Comment by smashwilson** [comment]
It's tough to set up package development on Windows! I couldn't get it working properly. so I had to poke around in the console and guess.

The first thing that I noticed is that you *don't* need the `.exe` extension, after all:

![no-exe-needed](https://cloud.githubusercontent.com/assets/17565/5810724/ed0ce7f2-a018-11e4-8bd2-e6a39a4eb97b.png)

The second is that environment variables, delimited with %, *are* interpolated, but not well:

![quoting-necessary](https://cloud.githubusercontent.com/assets/17565/5810732/081abc86-a019-11e4-94be-ff2effa93d67.png)

If you quote them, it works properly:

![windows-env-vars](https://cloud.githubusercontent.com/assets/17565/5810736/19e3e532-a019-11e4-95e8-955685b7bf42.png)

I can add your `%APPDATA%` path, then, and it's likely a better practice to use `%PROGRAMFILES%` for the path that's already there anyway. #109 takes care of both.

Thanks again for bringing this up! I don't use Atom on Windows often, so your feedback is most welcome :bow:
